sw/otbn/crypto/boot.s

```diff
@@ -0,0 +1,374 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * Unified boot-time OTBN program.
```

Comment:
```suggestion
 * Unified boot-services OTBN program.
```
?

Reply:
Changed the wording, but I'll just highlight here that there's one boot service this program can't provide for code-size reasons: ownership transfer, which requires ECDSA-P256 verification. There's no reason we couldn't include it, except that it won't fit in IMEM at the same time as attestation/sigverify. Using ECDSA-P256 for sigverify (or in general, using any one algorithm family for all boot services) would fix that.